tests/lib/c_lib/src/main.c

```diff
@@ -319,6 +322,206 @@ void test_bsearch(void)
 	zassert_not_null(result, "bsearch -key found");
 }
 
+/**
+ *
+ * @brief Test abs function
+ *
+ */
+
+void test_abs(void)
+{
+	int val = -5, value = 5;
+
+	zassert_equal(abs(val), 5, "abs -5");
+	zassert_equal(abs(value), 5, "abs 5");
+}
+
+/**
+ *
+ * @brief Test atoi function
+ *
+ */
+
+void test_atoi(void)
+{
+	char *str[6] = {"123", "2c5", "acd", "6 7a", "3-4e", "8+1c"};
+
+	zassert_equal(atoi(str[0]), 123, "atoi error");
```

Comment:
Please replace `str[0]` with `"123"` and same with all the test strings, so the test statements stand alone (don't have to look to see what `str[4]` means).  Also test the empty string.